### PR TITLE
⭐ Add Wally Publishing

### PR DESCRIPTION
**Pull request by medallyon**
Addresses #54 

This PR adds automated Wally publishing as an additional step to your GitHub Deployment Action.

There are a few steps you need to take to make sure this works correctly before merging this PR:

1. [Install Wally](https://wally.run/install)
2. Run `wally login`
  a. This will generate a file at `~/.wally/auth.toml` or `%userprofile%/.wally/auth.toml` on Windows
3. Copy (all of) the contents of this file and paste it into a new Repository Secret called `WALLY_AUTH` (as you have done for `ROBLOSECURITY`

### CODE REVIEW

File: wally.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[package]
+name = "1foreverhd/zoneplus"
```
Comment (medallyon) [inline wally.toml:2]: The lowercasing is a requirement by wally, and reports an error if corrected.

**Review by 1ForeverHD — APPROVED** [review]

**Comment by 1ForeverHD** [comment]
I've been putting off Wally so I appreciate this! I'll have this pulled in alongside wally support for TopbarPlus next week.

**Comment by medallyon** [comment]
@1ForeverHD Looks like an incorrect version of the `tarmac` dependency is preventing the wally Action to fail: https://github.com/1ForeverHD/ZonePlus/actions/runs/7750315703/job/21136340794

**Comment by 1ForeverHD** [comment]
Thanks will explore

**Pull request merged**